kas/include/tegra.yml

```diff
@@ -10,8 +10,6 @@ repos:
     refspec: kirkstone-l4t-r32.7.x
 
   meta-openembedded:
-    url: https://git.openembedded.org/meta-openembedded
-    refspec: kirkstone
```

Comment:
Nitpick: The commit only says that this is removed, but not why (and it's not clear to me).

Reply:
Very good catch. Addressed in update, okay with it?

Reply:
👍